src/simplification/Simplifier.ts

```diff
@@ -28,11 +35,19 @@ export default class Simplifier {
   simplifyRecursive(schema : Schema | boolean) : CommonModel[] {
     let models : CommonModel[] = [];
     let simplifiedModel = this.simplify(schema);
+    const containsAllTypes = (model : CommonModel) => {
+      if(model.type !== undefined){
+        if(Array.isArray(model.type)){
+          return model.type.length === 6;
+        }
+      }
+      return false;
+    };
     if(simplifiedModel.length > 0){
       //Get the root model from the simplification process which is the first element in the list
       const schemaSimplifiedModel = simplifiedModel[0];
       //Only if the schema is of type object and contains properties, split it out
-      if(schemaSimplifiedModel.type !== undefined && schemaSimplifiedModel.type.includes("object") && schemaSimplifiedModel.properties !== undefined){
+      if(schemaSimplifiedModel.type !== undefined && schemaSimplifiedModel.type.includes("object") && !containsAllTypes(schemaSimplifiedModel)){
```

Comment:
This actually solves a wrong interpretation for `test/processors/JsonSchemaInputProcessor/commonInputModel/combination_schemas.json` where object is not split out correctly.